.buildkite/scripts/steps/artifacts/docker_image.sh

```diff
@@ -98,14 +98,27 @@ tar -xf "kibana-$BASE_VERSION-cdn-assets.tar.gz" -C "$CDN_ASSETS_FOLDER" --strip
 gsutil -m cp -r "$CDN_ASSETS_FOLDER/*" "gs://$GCS_SA_CDN_QA_BUCKET/$GIT_ABBREV_COMMIT"
 gcloud auth revoke "$GCS_SA_CDN_QA_EMAIL"
 
+echo "--- Validate CDN assets"
+CDN_DESTINATION="https://kibana-cdn.gcp.qa.cld.elstc.co"
```

Comment:
Is there any scenario we need to override this based on the environment?

Reply:
I pushed https://github.com/elastic/kibana/pull/176943/commits/0929e517ee54d221c80792ef0581f60d1a40e333